### ISSUE TITLE
Get total tokens of a Java file

### ISSUE THREAD

**Issue by cdtpinto** [javaparser/javaparser#1579]
I want to parse a Java file and get all the Java tokens from it. For what I read [here](https://www.quora.com/What-are-tokens-in-java), Java tokens are basically all the variables, keywords, spcial characters, literals and operators.

So, what would be the best approach?

Should I walk the compilation unit detecting variables, keywords, etc... or JavaParser has an easier way to get the tokens from a compilation unit?

**Comment by cdtpinto** [comment]
Well, I think I found a solution using the `getTokenRange()` method.
If someone has the same problem in the future, this is the code I wrote:

```
File file = new File("FILE_LOCATION");
CompilationUnit cu = JavaParser.parse(file);
cu.getTokenRange().get().forEach(t -> System.out.println(t.getText()));
```

This prints every token in the console.

If someone has a better/prettier solution, I would gladly appreciate it.

However, I would like to ask a question to the maintainers of JavaParser. As far as I can see, you consider spaces and line breaks (`\n`) has Java tokens, am I right (this code I wrote prints them)? Are they really considered tokens?
